GOVERNANCE.md

```diff
@@ -0,0 +1,45 @@
+# Kyverno Governance
+
+This document defines governance policies for Kyverno project.
+
+- [Principles](#principles)
+- [Code of Conduct](#code-of-conduct)
+- [Meetings](#meetings)
+- [Roles and Process in the Kyverno Community](#roles)
+- [Conflict Resolutions](#conflict-resolutions)
+- [Changes](#changes)
+- [Credits](#credits)
+
+## Principles
+
+The Kyverno project community adheres to the following principles:
+
+- Open: The Kyverno community strives to be open, accessible and welcoming to everyone. Anyone may contribute, and contributions are available to all users according to open source values and licenses.
+- Transparent and accessible: Any changes to Kyverno source code and collaborations on the project are publicly accessible (GitHub issues, PRs, and discussions).
+- Merit: Ideas and contributions are accepted according to their technical merit and alignment with project objectives, scope, and design principles.
+
+
+## Code of Conduct
+
+Kyverno follows [Code of Conduct](./CODE_OF_CONDUCT.md), which is aligned with the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
+
+## Meetings
+
+Kyverno community meetings [schedule](https://kyverno.io/community/#community-meetings).
+
+Maintainers will also have closed meetings in order to discuss security reports or Code of Conduct violations. Such meetings should be scheduled by any Maintainer on receipt of a security issue or CoC report. All current Maintainers must be invited to such closed meetings, except for any Maintainer who is accused of a CoC violation.
```

Comment:
The word "maintainers" is not a proper noun in English. Aside from beginning a sentence, it should be in lower case.

Reply:
This has not been resolved.

Reply:
I rephrased it to "The maintainers..."

Reply:
But you have not addressed all instances of the word in this section.

Reply:
Resolved.